pkg/vault/test/authorizer/TimelockAuthorizer.test.ts

```diff
@@ -731,6 +734,192 @@ describe('TimelockAuthorizer', () => {
     });
   });
 
+  describe('addCanceler', () => {
+    context('when the sender is the root', () => {
+      let scheduledId = bn(0);
+      beforeEach('set sender', async () => {
+        from = root;
+      });
+
+      context('when creating canceler', () => {
+        context('for a specific scheduled execution id', () => {
+          it('can add canceler for a specific execution id', async () => {
+            expect(await authorizer.isCanceler(scheduledId, other)).to.be.false;
```

Comment:
`other` typically means 'an unrelated account that is not special in any way' (e.g. `expect(isCanceler(other)).to.be.false`). I'd create a `canceler` signer and use that.